minkyu/app.js

```diff
@@ -36,10 +39,40 @@ app.use(express.json());
 app.use(cors());
 app.use(morgan('tiny'));
 
-app.get("/ping", (req,res) =>{
-    res.json({message : "pong"});
+app.get("/pings", (req,res) =>{
+    res.status(201).json({"message" : "pong"});
 });
 
```

Comment:
안녕하세요 민규님 :) ping 엔드포인트는 단순 서버의 상태를 체크하는 용도로 활용되는 health check api로 복수(s)를 붙이지 않아도 될 것 같습니다 ㅎㅎ 또한 이는 단순 pong이라는 메세지를 받는 것에 불과하기 때문에 등록이나 수정을 나타내는 status code인 201이 아닌 성공적으로 반환되었음을 의미하는 200이 더욱 어울릴 것 같네요!
```suggestion
app.get("/pings", (req,res) =>{
    res.status(200).json({"message" : "pong"});
});

```

Reply:
네 알겠습니다! 처음에는 "/ping" 으로 썼는데 오류가 계속 나서 바꿔서 동작해보고 오류 안 나는거 확인하고서 다시 바꾼다는걸 깜빡했어요!! 주의하도록 하겠습니다!